roles/online_docs/templates/mkdocs/docs/R.md

```diff
@@ -17,27 +22,55 @@ touch ${HOME}/.Renviron
 This will create a ```.Renviron``` file in your home dir if you did not already have one.
 If the file already exists the touch command will only update the last modification time stamp of the file and leave the content unchanged.
 
-###### 2. Add ${R_LIBS_USER} to ${HOME}/.Renviron file
+#### 2. Add ${R_LIBS} to your ${HOME}/.Renviron file
 
-E.g. if the path you chose for your custom R package is in the ```tmp09``` folder of the group named ```my-favorite-group```,
+If the path you chose for your custom R package is in the ```tmp09``` folder of the group named ```my-favorite-group```
```

Comment:
Is er een reden om tmp09  te noemen, het kan op elke tmp toch?

Reply:
Het is gewoon een willekeurig voorbeeld. Ik zal eens kijken of ik er een variabele van kan maken, zodat je een tmp krijgt die beschikbaar is op het desbetreffende cluster....

Reply:
Getest met Talos -> tmp08 en Gearshift -> tmp01.